third_party/sphinx/LICENSE

```diff
@@ -0,0 +1,280 @@
+License for Sphinx
```

Comment:
We ended up splitting out happybase because it was under a different license:  is there any consideration to that here?  I think that `sphinx-ext-autosummary` will use these templates by default from within its own resources:  are we customizing them here?

Reply:
I believe it is okay in this case since we are putting it in a directory called `third_party`, which is different from the happybase case.

From some internal policies docs (go/thirdparty/non-google3):

> All non-Google code goes into the third_party/ directory of your project
>
> By default this should be a folder called third_party/ in the root directory of your project repository, but the exact name may vary by project.